.env

```diff
@@ -2,4 +2,11 @@ DATACITE_DOMAINS=*
 DATACITE_LOGIN=DATACITE.DCE
 DATACITE_PASSWORD=YOUR_PASSWORD_HERE
 DATACITE_PREFIX=10.24354
-SIDEKIQ_CONCURRENCY=5
\ No newline at end of file
+SIDEKIQ_CONCURRENCY=5
+# Version Information for footer
+GIT_SHA_DEV=$(git rev-parse HEAD 2> /dev/null)
```

Comment:
@little9, @bess It feels to me like easy variability across environments should be an advantage of using `ENV` here. Could we just do `GIT_SHA`, `GIT_BRANCH`, and `LAST_DEPLOYED`, then set the variables in deployment?

Sorry to be a bit of a pain about this one. :heart:

Reply:
@bess @no-reply 

I changed this so it's just referencing the variables. I'm not sure about the best way to set them in production. These return the correct values on the server:

```bash
GIT_SHA=$(tail -1 /opt/mahonia/revisions.log | awk '{ print substr($4, 1, length($4)-1) }' 2> /dev/null)
GIT_BRANCH=$(tail -1 /opt/mahonia/revisions.log | awk '{ print $2 }' 2> /dev/null)
LAST_DEPLOYED=$(tail -1 /opt/mahonia/revisions.log | awk '{ print $7 }' | date +%Y-%m-%d 2> /dev/null)
```